performant-software/DM
# Changed the order in which new viewers are added to the workspace

### What does this do?

1. Makes it so that newly opened windows appear to the right of existing ones, rather than to the left. Closes #25 
1. Removes Clean up project button from project viewer. Closes #39 
1. Also, replaces an incorrectly encoded jquery plugin file that prevented build. (jquery.event.drag-2.2.js). No related issue.

### How to test

1. Go to http://dm.performantsoftware.com
1. Log in
1. Open a project
1. Note that there is no longer a "Clean Up Project" button next to the "Done" button at bottom right of the project viewer dialog (re: issue #39)
1. Select a file to open from the list
1. Close the project viewer. 
1. Note location of the file.
1. Open the project again
1. Select another file to open from the list
1. Close the project viewer. 
1. Note second file opened to the right (re: issue #25)
1. Lather, rinse, repeat.
